Node/core/lib/dialogs/LuisRecognizer.js

```diff
@@ -28,7 +28,7 @@ var LuisRecognizer = (function (_super) {
     LuisRecognizer.prototype.onRecognize = function (context, callback) {
         var result = { score: 0.0, intent: null };
         if (context && context.message && context.message.text) {
-            var utterance = context.message.text;
+            var utterance = context.message.text.replace(/<at>.*<\/at>/g, '');
```

Comment:
This should also include other channels, and other methods of mentioning users and bots, like @